docs/design/mvp-summary-for-client.md

```diff
@@ -0,0 +1,65 @@
+# MVP Scope Summary for Client
+
+## Identity Access Control
+- User accounts
+  - manual creation, modification, and deletion for MVP
+- Staff roles
+  - manual creation, modification, and deletion for MVP
+- IP whitelist (can only access Chronicle from onsite at spa)
+
+## Customer Profiles
+- Core profiles field - creation and modification
+  - name
+  - address
+  - email
+  - phone number
+  - dob
+  - sex and gender
+- Medical record
+  - Weight and height
+  - Allergies
+  - Conditions
+  - Additional Notes
+- Membership level record
+  - Membership purchase and renewal history
```

Comment:
Is this MVP as well? I dont think it would be too difficult tbh just confirming.